airbyte-integrations/connectors/destination-iceberg/build.gradle

```diff
@@ -14,24 +14,24 @@ dependencies {
     implementation project(':airbyte-integrations:bases:base-java')
     implementation files(project(':airbyte-integrations:bases:base-java').airbyteDocker.outputs)
 
-    implementation('org.apache.spark:spark-sql_2.13:3.3.1') {
+    implementation('org.apache.spark:spark-sql_2.13:3.3.2') {
         exclude(group: 'org.apache.hadoop', module: 'hadoop-common')
     }
-    implementation('org.apache.spark:spark-hive_2.13:3.3.1') {
+    implementation('org.apache.spark:spark-hive_2.13:3.3.2') {
         exclude(group: 'org.apache.hadoop', module: 'hadoop-common')
     }
-    implementation 'org.apache.iceberg:iceberg-spark-runtime-3.3_2.13:1.1.0'
+    implementation 'org.apache.iceberg:iceberg-spark-runtime-3.3_2.13:1.3.0'
 
     // force awssdk version required by Iceberg
-    implementation "software.amazon.awssdk:utils:2.17.257"
-    implementation "software.amazon.awssdk:url-connection-client:2.17.257"
-    implementation "software.amazon.awssdk:s3:2.17.257"
-    implementation "software.amazon.awssdk:glue:2.17.257"
-    implementation "software.amazon.awssdk:dynamodb:2.17.257"
-    implementation "software.amazon.awssdk:kms:2.17.257"
-    implementation "software.amazon.awssdk:sts:2.17.257"
-    implementation "software.amazon.awssdk:sdk-core:2.17.257"
-    implementation "software.amazon.awssdk:aws-core:2.17.257"
+    implementation "software.amazon.awssdk:utils:2.20.18"
```

Comment:
this is to match the AWS SDK version being used by Iceberg: https://github.com/apache/iceberg/blob/apache-iceberg-1.3.0/versions.props#L24